sdk/src/main/java/io/opentelemetry/sdk/common/export/ConfigBuilder.java

```diff
@@ -139,4 +139,19 @@ protected static Long getLongProperty(String name, Map<String, String> map) {
       return null;
     }
   }
+
+  /**
+   * Get a {@link String} property from the map, {@code null} if it cannot be found.
+   *
+   * @param name The property name
+   * @param map The map where to look for the property
+   * @return the {@link String} value of the property, {@code null} if the property cannot be found.
+   */
+  @Nullable
+  protected static String getStringProperty(String name, Map<String, String> map) {
```

Comment:
do we have a unit test for this new method?

Reply:
I think we should add method for this.

Also this should just be map.get(). I am confused. https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#get-java.lang.Object-

Reply:
I'll provide tests for this in a separate PR with further cleanups :)